src/main/java/com/mcitmocks/mcitmocks/User/UserRepository.java

```diff
@@ -3,10 +3,12 @@
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.Optional;
+import java.util.UUID;
 
-public interface UserRepository extends CrudRepository<User, String> {
+public interface UserRepository extends CrudRepository<User, UUID> {
```

Comment:
If it hasn't been updated in `User`, we shouldn't update it here, probably? I don't mind which one we use, but we should be consistent.

Reply:
you are right. I will update from UUID to String